x/stakeibc/keeper/lsm_token_deposit.go

```diff
@@ -0,0 +1,51 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Stride-Labs/stride/v8/x/stakeibc/types"
+)
+
+func (k Keeper) SetLSMTokenDeposit(ctx sdk.Context, deposit types.LSMTokenDeposit) {
+	// TODO [LSM]
+}
+
+func (k Keeper) RemoveLSMTokenDeposit(ctx sdk.Context, denom string) {
+	// TODO [LSM]
+}
+
+func (k Keeper) GetLSMTokenDeposit(ctx sdk.Context, denom string) (deposit types.LSMTokenDeposit, found bool) {
+	// TODO [LSM]
+	return
+}
+
+func (k Keeper) GetAllLSMTokenDeposit(ctx sdk.Context) []types.LSMTokenDeposit {
+	// TODO [LSM]
+	return []types.LSMTokenDeposit{}
+}
+
+func (k Keeper) AddLSMTokenDeposit(ctx sdk.Context, deposit types.LSMDepositStatus) {
+	// TODO [LSM]
+	// See if a deposit already exists for this denom
+	// If so, increment the amount
+	// otherwise, create a new deposit
+}
+
+func (k Keeper) UpdateLSMTokenDepositStatus(ctx sdk.Context, deposit types.LSMDepositStatus, status types.LSMDepositStatus) {
+	// TODO [LSM]
+}
+
+func (k Keeper) GetLSMDepositsForHostZone(ctx sdk.Context, deposit types.LSMDepositStatus, chainId string) []types.LSMDepositStatus {
```

Comment:
Do we want a similar version of this function for validator as well to be used when rebalancing?

Reply:
If I recall correctly, we never have to grab the deposits for a specific validator (but I could be wrong).

I know for sure during rebalance, we want to grab them for the whole host zone.

Let's hold off on adding a validator version for now and we can always add it in in the future if we find that a function would use it